airflow/cli/commands/variable_command.py

```diff
@@ -76,7 +78,29 @@ def variables_import(args):
 
 def variables_export(args):
     """Exports all the variables to the file."""
-    _variable_export_helper(args.file)
+    var_dict = {}
+    with create_session() as session:
+        qry = session.query(Variable).all()
+
+        data = json.JSONDecoder()
+        for var in qry:
+            try:
+                val = data.decode(var.val)
+            except Exception:
```

Comment:
```suggestion
            except json.decoder.JSONDecodeError:
```

Reply:
It's always a good idea to handle specific exceptions as close to the source as possible.